bin/oio-meta2-rebuilder

```diff
@@ -19,77 +19,86 @@
 import argparse
 import sys
 
-from oio.rebuilder.meta2_rebuilder import Meta2Rebuilder
-from oio.common.logger import get_logger
+from oio.cli import make_logger_args_parser, get_logger_from_args
+from oio.directory.meta2_rebuilder import Meta2Rebuilder
 
 
 def make_arg_parser():
-    log_parser = argparse.ArgumentParser(add_help=False)
-    levels = ['DEBUG', 'INFO', 'WARN', 'ERROR']
-    log_parser.add_argument('--log-level', choices=levels,
-                            help="Log level")
-    log_parser.add_argument('--log-syslog-prefix',
-                            help="Syslog prefix")
-    log_parser.add_argument('--log-facility',
-                            help="Log facility")
-    log_parser.add_argument('--log-address',
-                            help="Log address")
-
-    descr = "Rebuild meta2 databases by setting 'last_rebuild' " \
-            "property in admin table, thus triggering a replication. " \
-            "And print the failed container IDs."
+    log_parser = make_logger_args_parser()
+    descr = """
+Rebuild meta2 databases by coppying the missing databases
+and synchronize this database.
```

Comment:
Rebuild meta2 databases by replacing the missing replicas and triggering a synchronization.